lib/api/html/functions.php

```diff
@@ -389,18 +389,22 @@ function beans_reset_markup( $id ) {
 }
 
 /**
- * Wrap markup.
+ * Register the wrap markup with Beans using the given markup ID.
  *
- * This function calls {@see beans_open_markup()} before the opening markup and {@see beans_close_markup()}
- * after the closing markup.
+ * This function registers an anonymous callback to the following action hooks:
+ *
+ *    1. `$id_before_markup`:  When this hook fires, {@see beans_open_markup()} is called to build the wrap's
+ *        opening HTML markup.
+ *    2. `$id_after_markup`: When this hook fires, {@see beans_close_markup()} is called to build the wrap's
+ *        closing HTML markup.
  *
  * Note: You can pass additional arguments to the functions that are hooked to <tt>$id</tt>.
  *
  * @since 1.0.0
  *
- * @param string       $id         The markup ID.
+ * @param string       $id         The wrap markup's ID.
  * @param string       $new_id     A unique string used as a reference. The $id argument may contain sub-hook(s).
- * @param string       $tag        The HTML wrap tag.
+ * @param string       $tag        The wrap's HTML tag.
  * @param string|array $attributes Optional. Query string or array of attributes. The array key defines the
  *                                 attribute name and the array value define the attribute value. Setting
  *                                 the array value to '' will display the attribute value as empty
```

Comment:
Do we need tests for setting the value to `'', false, or null` or are those "standard" behaviours?

Reply:
The tag by itself does not the current functionality of this function.  

That said, if there is no tag, then technically we should bail out.  Why? Nothing will get built, i.e. no HTML will be rendered for the wrap.   Thinking through that scenario, I don't think we run into much risk of adding a bail out.

I'll do that now.